static/js/hashchange.js

```diff
@@ -201,6 +201,18 @@ function do_hashchange_overlay(old_hash) {
 
     const coming_from_overlay = hash_util.is_overlay_hash(old_hash || "#");
 
+    if ((base === "settings" || base === "organization") && !section) {
+        let settings_panel_object = settings_panel_menu.normal_settings;
+        if (base === "organization") {
+            settings_panel_object = settings_panel_menu.org_settings;
+        }
+        history.replaceState(
+            null,
+            "",
+            get_full_url(base + "/" + settings_panel_object.current_tab()),
```

Comment:
I am not sure what to pass here for state and title parameters. As the [docs](https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState) says, title parameter is ignored by most browsers and anyways we do not show any different title when we open settings overlay.